### PR TITLE
Public connect project pages

### PR DESCRIPTION
**Pull request by Devorein**

### CODE REVIEW

File: apps/connect/middleware.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export async function middleware(request: NextRequest) {
   const user = session.user;
   const path = request.nextUrl.pathname;
 
-  if (!user && path !== '/') {
+  if (!user && path !== '/' && !path.startsWith('/projects/')) {
```
Comment (motechFR) [inline apps/connect/middleware.ts:15]: edge case: projects/new would fall into this case

Comment (Devorein) [reply apps/connect/middleware.ts:15]: Hmm good catch. Forgot about that one. We are going to replace projects path with /p anyways so will make that change

**Review by mattcasey — APPROVED** [review]

File: lib/farcaster/getFarcasterUsers.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  if (username) {
+    const farcasterUsersResponse = await GET<FarcasterUsersResponse>(
+      `${neynarBaseUrl}/user/search`,
+      {
+        q: username,
+        limit: 5
+      },
+      {
+        headers: {
+          Api_key: process.env.NEYNAR_API_KEY
```
Comment (mattcasey) [inline lib/farcaster/getFarcasterUsers.ts:61]: do u need to add this to the env config?

Comment (Devorein) [reply lib/farcaster/getFarcasterUsers.ts:61]: no this was added before

**Pull request merged**